test/unit/extent.js

```diff
@@ -144,37 +145,37 @@ describe('Extent', function () {
     });
 
     it('should clone tiled extent like expected', function () {
-        const withValues = new Extent('TMS:4326', zoom, row, col);
-        const clonedExtent = withValues.clone();
-        assert.equal(clonedExtent.zoom, withValues.zoom);
-        assert.equal(clonedExtent.row, withValues.row);
-        assert.equal(clonedExtent.col, withValues.col);
+        const withValues = new Tile('TMS:4326', zoom, row, col);
```

Comment:
Can you make a new file for Tile tests please?

Reply:
Agree, I was a bit lazy on this one. x)